### ISSUE TITLE
Cannot create wx.GraphicsContext from wx.msw.MetafileDC 

### ISSUE THREAD

**Issue by DietmarSchwertberger** [wxWidgets/Phoenix#811]
**Operating system**: Windows 10
**wxPython version**: 4.0.1
**Stock or custom build**: Stock
**Python version**: 3.6, 64 bit
**Stock or custom build**: stock

**Description of the problem**:
`GraphicsContext.Create` does not accept a MetafileDC as argument. Classic did.

```python
>>> import wx, wx.msw
>>> app = wx.App()
>>> dc = wx.msw.MetafileDC()
>>> gc = wx.GraphicsContext.Create(dc)
Traceback (most recent call last):
  Python Shell, prompt 9, line 1
builtins.TypeError: GraphicsContext.Create(): arguments did not match any overloaded call:
  overload 1: argument 1 has unexpected type 'MetafileDC'
  overload 2: argument 1 has unexpected type 'MetafileDC'
  overload 3: argument 1 has unexpected type 'MetafileDC'
  overload 4: argument 1 has unexpected type 'MetafileDC'
  overload 5: argument 1 has unexpected type 'MetafileDC'
  overload 6: too many arguments
```

The Classic code:
```python
>>> import wx
>>> app = wx.App()
>>> dc = wx.MetaFileDC()
>>> gc = wx.GraphicsContext_Create(dc)
```

**Comment by RobinD42** [comment]
Fixed by #845

**Comment by DietmarSchwertberger** [comment]
@RobinD42 : thanks a lot. I have now a prototype of an EMF backend for matplotlib running. It's working well, including clipping.
